src/NSrtm.Core/Pgm/DataDesciption/PgmDataDescriptionExtractor.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace NSrtm.Core.Pgm
+{
+    internal static class PgmPreamble
+    {
+        public static readonly IReadOnlyDictionary<string, Regex> Extractor =
+            new Dictionary<string, Regex>
+            {
+                {"MagicNumber", new Regex(@"^(P5)[^$]", RegexOptions.Multiline)},
```

Comment:
With option SingleLine he treates preamble like a one line. I changed to multiline by mistake.